go.mod

```
@@ -3,7 +3,7 @@ module github.com/devfile/library/v2
 go 1.19
 
 require (
-	github.com/devfile/api/v2 v2.2.1
+	github.com/devfile/api/v2 v2.2.2
 	github.com/devfile/registry-support/registry-library v0.0.0-20240311160550-e51ee8934746
```

There was some additional changes to registry-library that should be bumped to include in this devfile/library release:

```suggestion
	github.com/devfile/registry-support/registry-library v0.0.0-20240328155806-7c89891a72ce
```

---

These two commit specifically:
- https://github.com/devfile/registry-support/commit/9445a839c89f7a12336161c018a6d020b983840c
- https://github.com/devfile/registry-support/commit/c12332d7588042539f49e48fdfccdc877f2f332a

---

Good catch thanks :)